File: leetcode_1550/solution.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        counter = 0
        returnVal = False
        for item in arr:
            if (item % 2):
                counter = counter + 1
                if (counter == 3):
                    return True
            else:
                counter = 0
        return returnVal

File: leetcode_1550/solution.c
bool threeConsecutiveOdds(int* arr, int arrSize) {
    int counter = 0;
    bool returnVal= false;
    if(arrSize < 3)
    {
        return false;
    }
    for (int i = 0; i < arrSize; i++)
    {
        if((arr[i] % 2) != 0)
        {
            counter++;
            if(counter == 3)
            {
                return true;
            }
        }
        else
        {
            counter = 0;
        }
    }
    return returnVal;
}
